components/List.tsx

```diff
@@ -0,0 +1,14 @@
+interface Props {
+  lists: { content: string }[];
+}
+export function List({ lists }: Props) {
+  const className = "grid grid-cols-1 gap-1 list-decimal";
+  return (
+    <ul className={className}>
```

Comment:
**praise:** We should switch between <ul> and <ol> based on the prop `ordered` which is `boolean`.

Reply:
We should switch between what? I didnt see anything there.